packages/server/tests/helpers/utils.ts

```diff
@@ -150,4 +152,19 @@ export class Utils {
     }
     return res.toString();
   };
+
+  static ethCall = async (
+    relay: RelayClient,
+    callData: { from: string; to: any; gas: string; data: string },
+    requestId: string,
```

Comment:
Add block as a param, right now you've defaulted to `latest` with no way to modify

Reply:
Done.